template/provision.rb

```diff
@@ -0,0 +1,90 @@
+## Subspace configuration file
+#
+# This file contains all (most?) the configuration for provisioning servers to run this rails application.
+# It uses a simple DSL that should be easy to pick up if you're comfortable with things
+# like rails environments, devise config, capistrano, etc.
+#
+# When you run `subspace`, the following files be overwritten.  DO NOT MANUALLY MODIFY THESE FILES.
+#
+# config
+# - provision
+#   - group_vars/*
+#   - host_vars/*
+#   - ansible.cfg
+#   - hosts
+#   - playbook.yml
+
+# Step one: Configure your deployment environments.  We usually have a "dev" and a "production"
+# environment.  Sometimes we have a "staging" or qa.  You can do whatever you'd like.
+
+Subspace.configure do |config|
+  # Project level configuration
+
+  # Name of the project.  This will get used all over the place in path names, config file names, etc.
+  config.project_name = "<%= File.basename FileUtils.pwd =>"
```

Comment:
pwd = print working directory, this just gets the current directory name without the full path